### DISCUSSION TITLE
Ability to use Basic Auth

### DISCUSSION THREAD

**Discussion by jrich523** [ankitpokhrel/jira-cli#287] [Q&A] [answered]
It seems with the on-prem version we have you cant create access tokens. not sure if its disabled or just not possible, but we were told to use basic auth instead. 
would it be possible allow basic auth with this cli?

**Comment by ankitpokhrel** [comment, marked as answer]
@jrich523 the tool already supports Basic Auth. You can export the password you use for logging in as `JIRA_API_TOKEN`. See [instructions](https://github.com/ankitpokhrel/jira-cli#on-premise-installation).

**Comment by jrich523** [comment]
my apologies. i was searching the doc for "basic auth" or something specific to that. when i read the on-prem stuff i thought maybe it was just a language thing for the use of password vs token in that variable.

Thanks you, great work!
